amgut/db/0042.sql

```diff
@@ -0,0 +1,129 @@
+CREATE TABLE ag."source" (
+	source_id            uuid DEFAULT ag.uuid_generate_v4() ,
```

Comment:
`uuid_generate_v4` is a function in the `ag` schema? PostgreSQL is capable of generating those UUID.